tests/system/README.md

```diff
@@ -53,7 +53,14 @@ For example, to run only with the txtar files in/under `testdata/*foo*/*bar*`,
 including `testdata/*foo*/*bar*.txtar`:
 
 ```
-gotestsum -f testname -run /foo/bar
+gotestsum -f testname ./tests/system -run /foo/bar
+```
+
+To repeat a test `N` times when investigating flakiness, append the `-count`
+flag:
+
+```
+gotestsum -f testname ./tests/system -run /foo/bar -count N
```

Comment:
IMO document with `go test`, its more familiar.

Reply:
I understand, but I strongly prefer users to use `gotestsum -f testname` instead of `go test -v` for this purpose. The output is MUCH more useful this way. So I prefer to steer them in this way without having to think about replacing `go test` with `gotestsum`.